.github/workflows/test_formatting.yml

```diff
@@ -0,0 +1,25 @@
+name: Check Code Format
+on: [push]
+
+jobs:
+  code-black:
+    name: Check code formatting with Black and Flake8
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Install Black and flake8
+        run: pip install black==19.10b0 flake8
```

Comment:
Best case we can install these from a file like `requirements_dev.txt` instead of hardcoding